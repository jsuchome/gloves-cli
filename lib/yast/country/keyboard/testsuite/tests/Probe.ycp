/**
 * Test for Keyboard.ycp constructor behaviour in "installation"
 * Simulate that "dk" keyboard was set in linuxrc: see bug #118571
 *
 * Author:	Jiri Suchomel <jsuchome@suse.cz>
 * $Id: Probe.ycp 39978 2007-08-06 10:46:16Z jsuchome $
 */
{
    // testedfiles: Keyboard.ycp

    import "Testsuite";

    map READ = $[
	"sysconfig" : $[
	    "keyboard" : $[
		"YAST_KEYBOARD"	: "czech-qwerty,pc104",
	    ],
	],
	"target" : $[
	    "size"		: 1,
	    "dir"		: [],
	    "yast2"		: $[
		"czech-qwerty"	: [
		    "Czech (qwerty)",
		    $[
			"pc104": $[
				"ncurses": "cz-lat2-us.map.gz",
				"compose" : "latin2"
			],
		    ]
		],
		"danish"	: [
		    "Danish",
		    $[
			"pc104"	: $[
				"ncurses": "dk-latin1.map.gz"
			],
		    ],
		],
	    ],
	    "tmpdir"		: "/tmp",
	    "ycp"		: $[
		"XkbLayout"	: "cz_qwerty,us",
	    ],
	    "string"		: "",
	],
	"probe"	: $[
	    "architecture"	: "i386"
	],
	"etc"	: $[
	    "install_inf": $[
		"Keytable"	: "dk",
	    ],
	],
    ];
    map EXECUTE	= $[
	"target" : $[
	    "bash_output"	: $[],
	],
    ];

    Testsuite::Init([READ,$[],EXECUTE], nil);

    import "Stage";

    Stage::Set ("initial");

    import "Keyboard";
}
