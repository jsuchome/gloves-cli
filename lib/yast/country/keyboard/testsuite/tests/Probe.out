Read	.probe.keyboard nil
Read	.target.yast2 "lang2keyboard.ycp" $["czech-qwerty":["Czech (qwerty)", $["pc104":$["compose":"latin2", "ncurses":"cz-lat2-us.map.gz"]]], "danish":["Danish", $["pc104":$["ncurses":"dk-latin1.map.gz"]]]]
Read	.target.yast2 "keyboard_raw.ycp" $["czech-qwerty":["Czech (qwerty)", $["pc104":$["compose":"latin2", "ncurses":"cz-lat2-us.map.gz"]]], "danish":["Danish", $["pc104":$["ncurses":"dk-latin1.map.gz"]]]]
Read	.target.yast2 "keyboard_raw.ycp" $["czech-qwerty":["Czech (qwerty)", $["pc104":$["compose":"latin2", "ncurses":"cz-lat2-us.map.gz"]]], "danish":["Danish", $["pc104":$["ncurses":"dk-latin1.map.gz"]]]]
